client/components/jetpack/daily-backup-status/test/index.jsx

```diff
@@ -0,0 +1,110 @@
+/**
+ * @jest-environment jsdom
+ */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { shallow } from 'enzyme';
+import moment from 'moment';
+
+/**
+ * Internal dependencies
+ */
+import DailyBackupStatus from '..';
+
+import BackupFailed from '../status-card/backup-failed';
+import BackupScheduled from '../status-card/backup-scheduled';
+import BackupSuccessful from '../status-card/backup-successful';
+import NoBackupsYet from '../status-card/no-backups-yet';
+import NoBackupsOnSelectedDate from '../status-card/no-backups-on-selected-date';
+
+/**
+ * Mock dependencies
+ */
+jest.mock( 'react-redux', () => ( {
```

Comment:
This is a useful snippet that lets us mock any selector, but preserves the rest of Redux's real implementations.